src/elasticsearch/elasticsearch-sync.service.ts

```
@@ -0,0 +1,766 @@
+import { Injectable, Logger } from '@nestjs/common';
```

ESLint

[error] Replace `'@nestjs/common'` with `"@nestjs/common"`

(prettier/prettier)